test/plugins/cmd-returned-notify.plugin.bats

```diff
@@ -16,31 +19,33 @@ load ../../plugins/available/cmd-returned-notify.plugin
 
 @test "plugins cmd-returned-notify: do not notify before elapsed time" {
     export NOTIFY_IF_COMMAND_RETURNS_AFTER=10
-    export LAST_COMMAND_TIME=$(date +%s)
+    export COMMAND_DURATION_START_SECONDS="${EPOCHREALTIME:-$SECONDS}"
     sleep 1
     run precmd_return_notification
     assert_success
     assert_output $''
 }
 
 @test "plugins cmd-returned-notify: preexec no output" {
-    export LAST_COMMAND_TIME=
-    run preexec_return_notification
+    export COMMAND_DURATION_START_SECONDS=
+    run _command_duration_pre_exec
     assert_success
     assert_output ""
 }
 
 @test "plugins cmd-returned-notify: preexec no output env set" {
-    export LAST_COMMAND_TIME=$(date +%s)
-    run preexec_return_notification
+    skip "wut"
```

Comment:
wut

Reply:
I literally don't understand what this was supposed to test...maybe it doesn't exist after the refactoring?

Reply:
hmm. I am not sure myself. We should probably try to understand from the commit that added it, but if its just weird we should remove it entirely

Reply:
Ah ha! I think it's supposed to test that there's no output when the plug-in is disabled...like if someone enables the plug-in but then sets the duration variable to blank...but that's a silly thing to test for, so I think I'm just going to lose this test. It's identical to the test two steps up when the plug-in is enabled.

Reply:
Done and pushed.